### PR TITLE
Corrected an issue while syncing volume options

### PR DESCRIPTION
**Pull request by shtripat**
tendrl-bug-id: Tendrl/gluster-integration#105

Signed-off-by: Shubhendu <shtripat@redhat.com>

### CODE REVIEW

File: tendrl/gluster_integration/persistence/persister.py [hunk 1]
```diff
@@ -1,14 +1,25 @@
...
 class GlusterIntegrationEtcdPersister(EtcdPersister):
-    def __init__(self, config):
-        super(GlusterIntegrationEtcdPersister, self).__init__(config)
-        self._store = self.get_store()
+    def __init__(self):
+        etcd_kwargs = {
+            'port': int(config.get("commons", "etcd_port")),
+            'host': config.get("commons", "etcd_connection")
+        }
+        self.server = etcd_server(etcd_kwargs=etcd_kwargs)
```
Comment (r0h4n) [inline tendrl/gluster_integration/persistence/persister.py]: why are you calling it a "server"? Please fix later

File: tendrl/gluster_integration/persistence/persister.py [hunk 2]
```diff
@@ -1,14 +1,25 @@
+from tendrl.commons.config import load_config
+from tendrl.commons.etcdobj.etcdobj import Server as etcd_server
 from tendrl.commons.persistence.etcd_persister import EtcdPersister
 from tendrl.gluster_integration.persistence.sync_objects import SyncObject
 
+config = load_config(
+    "gluster-integration",
+    "/etc/tendrl/tendrl.conf"
+)
```
Comment (r0h4n) [inline tendrl/gluster_integration/persistence/persister.py]: This is going to fail, atleast check load_config signature first

Comment (shtripat) [inline tendrl/gluster_integration/persistence/persister.py]: I feel this PR we should look at once refactoring related to config is in.

**Comment by r0h4n** [comment]
Fix travis failures @shtripat

**Comment by shtripat** [comment]
Taken care as part of https://github.com/Tendrl/gluster-integration/pull/117

**Pull request closed without merging**